pattern-2/tile/tile.yml

```diff
@@ -47,12 +60,25 @@ forms:
   label: AM DB connection information
   description: WSO2 API Manager Database connection information
   properties:
-  - name: am_db_hostname
+  - name: am_db_jdbc_url
     type: string
-    label: Hostname
-  - name: am_db_name
+    label: JDBC URL
+  - name: am_db_driver
+    type: dropdown_select
+    label: Driver Class Name
+    options:
+    - name: com.mysql.jdbc.Driver
+      label: com.mysql.jdbc.Driver
+      default: true
+    - name: com.microsoft.sqlserver.jdbc.SQLServerDriver
+      label: com.microsoft.sqlserver.jdbc.SQLServerDriver
+    - name: postgres
+      label: postgres
+    - name: oracle.jdbc.driver.OracleDriver
```

Comment:
If someone selects this, it won't work as the bosh deployment does not support this for the moment. Shall we remove this?

Reply:
fixed with https://github.com/wso2/pivotal-cf-apim/pull/49/commits/328ee6d6be3730567554d39a86a2c2829dd01c5c